doc/api/url.md

```diff
@@ -1151,13 +1151,22 @@ console.log(url.domainToUnicode('xn--iñvalid.com'));
 // Prints an empty string
 ```
 
-### `url.fileURLToPath(url)`
+### `url.fileURLToPath(url[, options])`
 
 <!-- YAML
 added: v10.12.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/52509
+    description: The `options` argument can now be used to
+                 determine how to parse the `path` argument.
 -->
 
 * `url` {URL | string} The file URL string or URL object to convert to a path.
+* `options` {Object}
+  * `windows` {boolean|undefined} `true` if the `path` should be
+    treated as a windows filepath, `false` otherwise.
+    **Default:** `undefined`.
```

Comment:
The implementation suggests that by default (if `windows` is `undefined` or `null`), the function will fallback to old behaviour (autodetection, depending on platform). It's probably worth mentioning here.

Also nit: "if `path` should be treated as windows filepath" can be reworded to "returned path should be a windows filepath"